src/components/MdTable/MdTable.vue

```diff
@@ -215,7 +215,9 @@
       return { MdTable }
     },
     watch: {
-      value() {}, // Watch this prop to make the table reactive to props
+      value() {
+        if (isPromise(this.value)) this.resolvePromise()
```

Comment:
you need to implement `else` too because all other tables that doesnt use promise are broken.

Reply:
I think better is to make it as computed

```
items () {
  if (isPromise(this.value)) {
    return this.resolvePromise()
  } else {
    return this.value
  }
}
```

Reply:
There shouldn't be a mutation in a computed.

Reply:
Thanks guys. Care to share the secret of how you guys are so sharp?

@Samuell1 I think that example doesn't work because `this.resolvePromise` doesn't (and cannot?) return the resolved value.

Reply:
@emman27 It will work like `then((resolve) => (resolve))`, but @VdustR said it will be needed to change resolvePromise method.

I think this will now work.